packages/webapp/src/components/FullYearCalendar/index.jsx

```diff
@@ -9,6 +9,7 @@ import { getNewDate } from '../Form/InputDuration/utils';
 import 'rc-year-calendar/locales/rc-year-calendar.es';
 import 'rc-year-calendar/locales/rc-year-calendar.pt';
 import 'rc-year-calendar/locales/rc-year-calendar.fr';
+import 'rc-year-calendar/locales/rc-year-calendar.de';
```

Comment:
I guess we need a PR to either find a new Calendar package or add these files to our locales then copy them to node_modules to augment package.